bin/improver-percentile

```diff
@@ -71,6 +71,11 @@ def main():
                         "when converting probabilities to percentiles and may "
                         "be omitted. This coordinate(s) will be removed "
                         "and replaced by a percentile coordinate.")
+    parser.add_argument('--ecc_bounds_warning', default=False,
+                        action='store_true',
+                        help='Option to specify that exceeding ECC bounds '
+                             'will only result in a warning message rather '
+                             'than an exception.')
```

Comment:
Can we be a bit more explicit, eg: "If True, where calculated percentiles are outside the ECC bounds range, raise a warning rather than an exception."

Reply:
Changed.